### DISCUSSION TITLE
Survey builder: Es un bug que el sistema no te valide los long_text al momento de guardar en un formulario ya existente?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#199] [2. Questions and Answers] [answered]
Hola

Dejo el link del tema anterior para no abrir el ya existente:
https://github.com/Cotalker/documentation/discussions/126

Hola @gmoplaza por casualidad está subido el fix que evita ese error?- Te lo comento porque en las implementaciones existentes, debo por base de dato a cambiar el subtipo a short_text para que pueda hacer cambios en el formulario y me gustaría saber si se subirá pronto :>

Gracias

**Comment by acedward** [comment, marked as answer]
Si aun está fallando por favor, contactar tu representante Cotalker para dar prioridad al bug.
Gracias!
